src/functionalTest/java/uk/gov/hmcts/reform/sendletter/FunSuite.java

```diff
@@ -17,5 +17,8 @@
     @BeforeClass
     public static void setUp() {
         variables.set("APPLICATION_INSIGHTS_IKEY", "some-key");
+        variables.set("servicebus.connectionString", "url");
+        variables.set("servicebus.queue.name", "send-letter-queue");
+        variables.set("servicebus.queue.messageTTLInDays", "7");
```

Comment:
unlike `APPLICATION_INSIGHTS_IKEY` this config entries don't need to be set as env variables for test